aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -59,11 +65,13 @@ public void processDailyEmail(Instant scheduledFireTime) {
         }
     }
 
-    List<AggregationCommand> processAggregateEmails(Instant scheduledFireTime) {
-        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(DAILY.getDuration());
+    List<AggregationCommand> processAggregateEmails() {
+        Instant scheduledFireTime = emailAggregationResources.getLastCronJobRun().getLastRun();
+        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(EmailSubscriptionType.DAILY.getDuration());
 
         LocalDateTime endTime = LocalDateTime.ofInstant(scheduledFireTime, UTC);
         LocalDateTime startTime = LocalDateTime.ofInstant(yesterdayScheduledFireTime, UTC);
```

Comment:
These conversions could be avoided if `CronJobRun#lastRun` was a `LocalDateTime` instead of an `Instant`.